src/app/records/Record.m.scss

```diff
@@ -62,12 +71,78 @@
   position: relative;
   border-color: #f37423;
   outline: 1px solid #f37423;
+  background: transparent;
+}
+
+.gildingTriumph {
+  background-image: linear-gradient(
+      to left bottom,
+      transparent 25%,
+      scale-color(#a1a2a2, $alpha: -80%) 1px,
+      transparent calc(25% + 1px)
+    ),
+    linear-gradient(
+      to left top,
+      transparent 25%,
+      scale-color(#a1a2a2, $alpha: -80%) 1px,
+      transparent calc(25% + 1px)
+    );
+  background-size: 60px 100%;
+  // The following offset and repeat settings
+  // prevent the two gradients making up the chevron pattern
+  // (which both use reduced opacity colors) from overlapping
+  // at the 0 point, which prevents that one point from being
+  // more saturated than the rest of the lines.
+  // There is no visible gap.
+  background-position: 0 0, 0 2px;
+  background-repeat: repeat-x;
+}
+
+.redeemed.gildingTriumph {
+  border-color: $gilded-triumph-border-color;
+  background-image: linear-gradient(
+      to left bottom,
+      transparent 25%,
+      scale-color($gilded-triumph-border-color, $alpha: -80%) 1px,
+      transparent calc(25% + 1px)
+    ),
+    linear-gradient(
+      to left top,
+      transparent 25%,
+      scale-color($gilded-triumph-border-color, $alpha: -80%) 1px,
+      transparent calc(25% + 1px)
+    );
+}
+
+.glow {
+  display: none;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.trackedInDim .glow {
+  display: block;
   background: linear-gradient(
     scale-color($orange, $alpha: -100%) 0%,
     scale-color($orange, $alpha: -80%) 100%
   );
 }
 
+.redeemed.gildingTriumph .glow {
+  display: block;
+  background: linear-gradient(
+    scale-color($gilded-triumph-border-color, $alpha: -100%) 80%,
+    scale-color($gilded-triumph-border-color, $alpha: -90%) 100%
+  );
+}
+
+.redeemed .gildingText {
+  opacity: 0.5;
```

Comment:
Similar to above comment re: gilding text colors, this opacity dim matches the opacity dim of a completed set of objectives for a triumph. Since rewards "disappear" after a triumph is complete, I wanted a way to show that this was "complete" as well, so I decided to match the styling of the objectives. Happy to change if people feel differently.